src/pages/DetaljerPage.tsx

```diff
@@ -29,8 +28,11 @@ const DetaljerPage = () => {
     if (showAlert) setTimeout(() => setShowAlert(false), 10000);
   }, [showAlert]);
 
-  if (!oppdragsId) window.location.replace(BASENAME);
+  if (!location.state) {
```

Comment:
Skulle ikke dette vekk til fordel for zustand?

Reply:
fiksa